tensorflow/python/framework/importer.py

```diff
@@ -151,7 +151,8 @@ def _FindAttrInOpDef(attr_name, op_def):
 
 def import_graph_def(graph_def, input_map=None, return_elements=None,
                      name=None, op_dict=None, producer_op_list=None):
-  """Imports the TensorFlow graph in `graph_def` into the Python `Graph`.
+  """Imports the TensorFlow graph in `graph_def` into the current default Python
+  `Graph`.
```

Comment:
Can you make this fit in 80 characters? I suggest either: 
- Remove the somewhat redundant 'Python' and 'TensorFlow'
or
- Put a full sentence about how the graph is added to the default graph into the main text for this function.

or both.